modules/SiteHeader/assets/scripts/main.mjs

```diff
@@ -0,0 +1 @@
+import './has-sticky-header.js';
```

Comment:
I think it's a better convention to import register functions from supporting files and then do the actual register call in the main file. Look at core's UIFramework module for this pattern. In this way side effects aren't triggered simply by importing a js module file.